R/algorithm.R

```diff
@@ -226,3 +221,18 @@ get_algorithm_logic <- function(logic_name, algorithm = NULL) {
       "stringr::str_detect(\\1, '\\2')"
     )
 }
+
+#' Parse the logic strings into R expressions.
+#'
+#' @param logic The name of the logic to use.
+#'
+#' @return An R expression.
+#' @keywords internal
+#'
+logic_as_expr <- function(logic) {
```

Comment:
```suggestion
logic_as_expression <- function(logic) {
```
To avoid abbreviations? :)